test/data/02/README.md

```diff
@@ -14,5 +19,3 @@ $$f(s_1, s_3, s_5) = 2.7 \left( 1.23 + 1.3 s_1 - 0.7 s_3 - 0.2 s_1 s_2 + 1.5 s_1
 | MiniZinc |  Spin  |   ❌    |
 | QUBO     |  Bool  |   ✔️    |
 | Qubist   |  Spin  |   ✔️    |
-
-[symbols]: # (✔️❌)
```

Comment:
There is a reason to have this legend here

Reply:
Not really